kpops/utils/docstring.py

```diff
@@ -21,29 +21,39 @@ def describe_attr(name: str, docstr: str | None) -> str:
     return _trim_description_end(docstr)
 
 
-def describe_class(docstr: str | None) -> str:
-    """Return class description from its docstring
+def describe_object(docstr: str | None) -> str:
+    """Return free-text description from a docstring
```

Comment:
Return description from an object's docstring

Reply:
not sure what free-text means tbh

Reply:
Now I'm uncertain, too. Might not be correct in English

Anyway, I replaced it with yours, it's simpler